### PR TITLE
Create Blog “build-your-own-ilo-redfish-simulator”

### PR DESCRIPTION
**Pull request by donzef**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/build-your-own-ilo-redfish-simulator.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+```shell
+python3.4 redfishMockupCreate.py -r ilo5 -u ilouser -p ilopassword \
+     --Secure --Auth Session  --Headers \
+     --Dir ./ilo5
+```
+
+### Preparing the use of iLOrest against your Mockup Server
+
+If you intend to use [ilOrest](http://hpe.com/info/resttool) against the mockup you created with the above command, you should, just after the mockup creation, open an iLOrest session and capture its cache. This cache directory is created during the authentication process in a default location unless a specific location is specified on the command line.
```
Comment (pattdepanick) [inline content/blog/build-your-own-ilo-redfish-simulator.md:57]: replace just by right as  in right after

File: content/blog/build-your-own-ilo-redfish-simulator.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+python3.4 redfishMockupCreate.py -r ilo5 -u ilouser -p ilopassword \
+     --Secure --Auth Session  --Headers \
+     --Dir ./ilo5
+```
+
+### Preparing the use of iLOrest against your Mockup Server
+
+If you intend to use [ilOrest](http://hpe.com/info/resttool) against the mockup you created with the above command, you should, just after the mockup creation, open an iLOrest session and capture its cache. This cache directory is created during the authentication process in a default location unless a specific location is specified on the command line. 
+
+The easiest to perform this action is to install [iLOrest](https://github.com/HewlettPackard/python-redfish-utility/releases/latest) on your favorite operating system and identify the default cache location with the `help` command.
```
Comment (pattdepanick) [inline content/blog/build-your-own-ilo-redfish-simulator.md:59]: The easiest way to perform this action

**Comment by donzef** [comment]
Fixed the "Easiest way" typo

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Francois, 
If you want to have the blog post tied to the ILO RESTFul platform page in HPE DEV portal, I would suggest you to add a Tag: ilo-restful-api 

Denis

**Review by choukden — APPROVED** [review]

File: content/blog/build-your-own-ilo-redfish-simulator.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+On a Linux system, this operation can be done with the following commands:
+
+```Shell
+cd $iloCacheDir/cache/
+sed -i 's?\("url": "https://\)ilo-IP-physical"?\1ilo-IP-simulator"?' *
+```
+Once the iLOrest cache points to your mockup server, you can use this Redfish client tool to query the mockup. 
+
+> **NOTE** If you use the `login` iLOrest commands, the cache will be overwritten. If you use the `logout` command, the cache will be erased.
```
Comment (drensing) [inline content/blog/build-your-own-ilo-redfish-simulator.md:140]: In the previous note, you did not bold "NOTE" and you had a colon after it. You probably want to retain consistency, so pick one way or the other. I recommend, at least, placing a colon after "Note".

File: content/blog/build-your-own-ilo-redfish-simulator.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+## Conclusion
+
+To learn and test the Redfish API, The DMTF provides two very useful tools to create Redfish mockups and simulate a live service. They are easy to install and use and their quality is a good. In addition, from my personal experience, I found the maintainers of these active GithHub projects very responsive to address quality issues and proposed enhancements.
```
Comment (drensing) [inline content/blog/build-your-own-ilo-redfish-simulator.md:144]: "The DMTF" should be "the DMTF"
Place a comma after "They are easy to install and use" in order to set it apart from the next thought, which is "and their quality is good."
Also, remove the "a" before "good".
"GithHub" should probably be "GitHub"

**Review by drensing — CHANGES_REQUESTED** [review]

File: content/blog/build-your-own-ilo-redfish-simulator.md (hunk 1 above)
Comment (donzef) [reply content/blog/build-your-own-ilo-redfish-simulator.md:58]: Done.

File: content/blog/build-your-own-ilo-redfish-simulator.md (hunk 2 above)
Comment (donzef) [reply content/blog/build-your-own-ilo-redfish-simulator.md]: Done

**Pull request merged**